components/alert/alert.css

```diff
@@ -0,0 +1,54 @@
+@import url("/index.css");
```

Comment:
For local files following will work.
@import "/index.css";

Not sure if using URL will have a performance impact or not, so please read about it.

Reply:
Okay . I'll read about it and update in what will correct way to import file.